.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -25,7 +25,7 @@ Optionally add one or more of the following kinds if applicable:
 
 **Does this PR introduce a user-facing change? Then add your Release Note here**:
 <!--
-Write your release note:
+Write your release note. Release notes are being used to generate the Changelog:
```

Comment:
Uber nit: Changelog should be "changelog", it's not a title or something else that warrants a capital letter, just a normal noun.

Reply:
that is a good point. I throught write it capitalized because of "the", but turns out that is not the case in English for regular nouns. Fixed